### DISCUSSION TITLE
Select all but password

### DISCUSSION THREAD

**Discussion by SupertigerDev** [prisma/prisma#14837] [Q&A] [answered]
Is there anyway to select all fields apart from the `password` one? It doesn't seem to let me do it in `include`
![image](https://user-images.githubusercontent.com/8193841/184961176-1dac2961-9da4-4b00-9069-548afc756d32.png)

**Comment by austincrim** [comment, marked as answer]
Hey @Supertigerr 👋,

It's not possible to exclude fields by name yet, though there is an [outstanding feature request](https://github.com/prisma/prisma/issues/5042) that we are looking at.

The workarounds so far are to list every other field in the `include` or write a [middleware](https://www.prisma.io/docs/concepts/components/prisma-client/middleware) that removes the unwanted field for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
